tasks/styles.js

```diff
@@ -1,20 +1,24 @@
 'use strict';
 
+const getOptions = require('../lib/getOptions');
+const registerTaskWithProductionMode = require('../lib/registerTaskWithProductionMode');
+
 module.exports = (gulp, $, pkg) => {
   const reportError = require('../lib/error.js');
 
   // Copyright notice placed at top of compiled CSS
   const copyrightPlaceholder = '/*! #copyright DO NOT REMOVE# */';
-  const copyrightNotice = ['/*!',
-    ' * ' + pkg.title + ' - ' + pkg.description,
-    ' * @version v' + pkg.version,
-    ' * @link ' + pkg.homepage,
-    ' * @author ' + pkg.author,
+  const copyrightNotice = [
+    '/*!',
+    ` * ${pkg.title} - ${pkg.description}`,
+    ` * @version v${pkg.version}`,
+    ` * @link ${pkg.homepage}`,
+    ` * @author ${pkg.author}`,
     ' */',
-    ''].join('\n');
+    '',
+  ].join('\n');
 
-  // Settings for clean-css plugin
-  const cleanCssOptions = {
+  const cleanCssPluginOptions = {
```

Comment:
Why is the renaming necessary?

Reply:
It's not strictly necessary, but I though the comment was superfluous, removed it and wanted to be explicit on the variable name in return.